packages/eslint-config/react.js

```diff
@@ -0,0 +1,66 @@
+const { resolve } = require('node:path');
+
+const project = resolve(process.cwd(), 'tsconfig.json');
+
+/*
+ * This is a custom ESLint configuration for use a library
+ * that utilizes React.
+ *
+ */
+
+module.exports = {
+  env: {
+    es6: true,
+    browser: true,
+    node: true,
+    jest: true
+  },
+  extends: ['eslint:recommended', 'plugin:import/errors', 'plugin:import/warnings', 'plugin:react/recommended'], //, 'plugin:react-hooks/recommended'],
```

Comment:
We should probably include this 2 as well 
"plugin:@typescript-eslint/eslint-recommended",
    "plugin:@typescript-eslint/recommended"